src/scss/_contact.scss

```diff
@@ -1,87 +1,78 @@
 .contact {
-  background: $bg-color;
   color: $heading-color;
-  padding: 5rem 0;
+  padding: 50px 0;
+  text-align: center;
   width: 100%;
 
-  .contact-box {
-    margin: 0 auto;
-    max-width: 65rem;
-    text-align: center;
-    width: 80%;
+  .contact-heading {
+    font-size: 2.6rem;
+  }
 
-    .contact-heading {
-      font-size: 2.6rem;
+  .divider {
+    margin: 20px auto 30px;
+  }
 
-      &::after {
-        background: $akai-yellow;
-        content: '';
-        display: block;
-        height: 0.3rem;
-        margin: 2.5rem auto;
-        width: 4.5rem;
-      }
-    }
+  .contact-info {
+    @include font-size(1.3);
 
-    .contact-info {
-      color: $text-color;
-      font-size: 1.5rem;
-      line-height: 2.6rem;
-      margin: 1.5rem 3rem 4rem;
-    }
+    color: $text-color;
+    line-height: 2.6rem;
+    margin: 0 auto 30px;
+    max-width: 520px;
+  }
 
-    .contact-form {
-      display: block;
-      margin: 0 auto 2.5rem;
-      max-width: 35rem;
+  .contact-form {
+    display: block;
+    margin: 0 auto 30px;
+    max-width: 350px;
 
-      .email,
-      .message {
-        background: transparent;
-        border: 0;
-        border-bottom: $border;
-        box-sizing: border-box;
-        color: $text-color;
-        font-family: inherit;
-        font-size: 1.5rem;
-        opacity: 0.8;
-        padding: 1.5rem;
-        width: 100%;
+    .email,
+    .message {
+      @include font-size(1.3);
 
-        &:focus {
-          outline: none;
-        }
+      background: transparent;
+      border: 0;
+      border-bottom: $border;
+      box-sizing: border-box;
+      color: $text-color;
+      font-family: inherit;
+      opacity: 0.8;
+      padding: 12px;
+      transition: all 0.3s ease-out;
+      width: 100%;
 
-        &.disable:invalid {
-          border-color: $border-invalid;
-        }
+      &:focus {
+        outline: none;
       }
 
-      .message {
-        margin: 1.8rem 0;
-        resize: none;
+      &.disable:invalid {
+        border-color: $invalid;
       }
+    }
+
+    .message {
+      margin: 20px 0 30px;
+      resize: none;
+    }
 
-      .send-btn {
-        background: $akai-yellow;
-        border: 0;
-        border-radius: 25px;
-        box-shadow: 0 5px 30px -5px #aaa;
-        color: #fff;
-        font-family: inherit;
-        font-size: 1.4rem;
-        font-weight: bold;
-        margin-top: 1.5rem;
-        padding: 1.2rem 1rem;
-        transition: all, 0.4s, ease-out;
-        width: 22rem;
+    .send-btn {
+      @include font-size(1.4);
 
-        &:hover {
-          background: darken($akai-yellow, 6%);
-          color: #fff;
-        }
-      }
+      background: $akai-yellow;
+      border: 0;
+      border-radius: 25px;
+      box-shadow: 0 5px 30px -5px $shadow;
+      color: $white-text;
```

Comment:
Would $white-text be different than $white-background, I think we should use something like $akai-white

Reply:
Yes, these colors differ.